libs/design-system/brand-logo/src/lib/brand-logo.component.html

```diff
@@ -0,0 +1,10 @@
+<a
+  href="https://humanatlas.io/"
+  target="_blank"
+  [inlineSVG]="'assets/logo/hra-brand-logo.svg'"
+  [evalScripts]="NEVER_EVAL_SCRIPTS"
+  aria-labelledby="brand-name"
```

Comment:
There is not element with id `brand-name`, please change to `aria-label="Visit Human Atlas"`